TACC/Core-Styles
# fix(demo): no cross-pattern demo asset load

## Overview / Changes

Move a cross-pattern style from `tools/x-link/demo.css` to `core-styles.demo.css`.

## Related

- required by https://github.com/TACC/Core-CMS/pull/569

## Testing

1. `npm run build && npx serve demo`
2. Open "Elements" > "Links".
3. Verify paragraphs are not more than ~50ch long.
4. Open "Trumps" > "Links (Irregular)".
5. Verify paragraphs are not more than ~50ch long.

## UI

| Elements | Trumps
| - | -
| ![Elements](https://user-images.githubusercontent.com/62723358/206498918-b8656dc0-268c-4d2b-b9cc-894422435e4f.png) | ![Trumps](https://user-images.githubusercontent.com/62723358/206498915-741dfbc3-adc3-40aa-b49b-6d17aa927741.png)
